payjoin/src/uri/mod.rs

```diff
@@ -159,8 +159,8 @@ impl bitcoin_uri::de::DeserializationState<'_> for DeserializationState {
             "pj" => Err(InternalPjParseError::DuplicateParams("pj").into()),
             "pjos" if self.pjos.is_none() => {
                 match &*Cow::try_from(value).map_err(|_| InternalPjParseError::BadPjOs)? {
-                    "0" => self.pjos = Some(false),
-                    "1" => self.pjos = Some(true),
+                    "0" => self.pjos = Some(true),
+                    "1" => self.pjos = Some(false),
```

Comment:
Upon closer inspection, I think changing this as well as the `SerializeParams` is semantically backwards. 0 should still be false and 1 true for pjos, but pjos == !disable_output_substitution.

This will require another change: `disable_output_substitution: !pjos.unwrap_or(true),` in

```rs
                    Ok(MaybePayjoinExtras::Supported(PayjoinExtras {
                        endpoint,
                        disable_output_substitution: !pjos.unwrap_or(true),
                    }))
```